doc/source/cluster/kubernetes/troubleshooting/troubleshooting.md

```diff
@@ -7,13 +7,23 @@ If you don't find an answer to your question here, please don't hesitate to conn
 
 # Contents
 
+- [Use ARM-based docker images for Apple M1/M2 Macbooks](#docker-image-for-apple-macbooks) 
 - [Upgrade KubeRay](#upgrade-kuberay)
 - [Worker init container](#worker-init-container)
 - [Cluster domain](#cluster-domain)
 - [RayService](#rayservice)
 - [Autoscaler](#autoscaler)
 - [Other questions](#other-questions)
 
+(docker-image-for-apple-macbooks)=
+## Use ARM-based docker images for Apple M1/M2 Macbooks
+Currently Ray builds different images for different platforms. Until Ray moves to building multi-architecture images, [tracked by this Github issue](https://github.com/ray-project/ray/issues/39364), the solution here is to use platform specific docker images in the head and worker group specs of the [RayCluster config](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/config.html#image). 
+
+Use an image with the tag `aarch64` (e.g. `image: rayproject/ray:2.12.0-aarch64`) if you are running KubeRay on a Mac M1 or M2.
```

Comment:
can we change this to `2.20.0`?

also, this is assuming user has some form of docker installed on the mac (which is often based on a Linux virtual machine)? Should you mention how to install docker?

Reply:
> Should you mention how to install docker?

I think we may not need to mention it at this moment.

Reply:
updated to `2.20.0`, thanks for the review.